giles-eco/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -30,6 +33,11 @@
         <property name="fallbackToSystemLocale" value="false" />
     </bean>
 
+    <bean id="uiMessages"
+        class="org.springframework.beans.factory.config.PropertiesFactoryBean">
+        <property name="location" value="classpath:locale/messages.properties" />
+    </bean>
+
```

Comment:
there is already  amessage source defined, why don't you use that one instead of loading the same file again?